src/Stepper/StepConnector.js

```diff
@@ -0,0 +1,53 @@
+import React, {PropTypes} from 'react';
+import pure from 'recompose/pure';
+
+const StepConnector = (props, context) => {
+  const {muiTheme, stepper} = context;
+
+  const styles = {
+    wrapper: {
+      flex: '1 1 auto',
+    },
+    line: {
+      display: 'block',
+      borderColor: muiTheme.stepper.connectorLineColor,
+    },
+  };
+
+  /**
+   * Clean up once we can use CSS pseudo elements
+   */
+  if (stepper.orientation === 'horizontal') {
+    styles.line.marginLeft = -6;
+    styles.line.borderTopStyle = 'solid';
+    styles.line.borderTopWidth = 1;
+  } else if (stepper.orientation === 'vertical') {
+    styles.wrapper.marginLeft = 14 + 11; // padding + 1/2 icon
+    styles.line.borderLeftStyle = 'solid';
+    styles.line.borderLeftWidth = 1;
+    styles.line.minHeight = 28;
+  }
+
+  const {prepareStyles} = muiTheme;
+
+  return (
+    <div style={prepareStyles(styles.wrapper)}>
+      <span style={prepareStyles(styles.line)}></span>
+    </div>
+  );
+};
+
+StepConnector.propTypes = {
```

Comment:
For some reason I thought we had an eslint rule that made sure propTypes and contextTypes were declared at top of file (so that it is more consistent in presentation to a class).

I don't have a strong preference, but I think we're following that convention in our other functional components.